File: labaTRPOtransport/Implement/transport/Tram.java
package com.company.transport;

import com.company.Person.Blogger;
import com.company.Person.Driver;
import com.company.Person.Passanger;
import com.company.Route;

import java.util.ArrayList;

public class Tram extends GroundTransport {
    public Tram(){}
    public Tram(int number, int amount, int[] timeTable, Route route, Driver driver, ArrayList<Passanger> passangers, Blogger blogger){
        super(number, amount, timeTable, route, driver, passangers, blogger);
        this.setName("Tram");
    }

    @Override
    public void go() {
        System.out.println("Tram Work");
    }
}

File: lab3/FIrstUMLLab/Tram.cs
using System;
using System.Collections.Generic;

namespace FIrstUMLLab
{
    class Tram : GroundTransport
    {
        public Tram(Driver driver, List<DateTime> timetable, int amount, int stock)
            : base(driver, timetable, amount, stock)
        { }
    }
}
